src/site/content/en/payments/web-based-payment-apps-overview/index.md

```diff
@@ -92,10 +92,9 @@ if used in a top-level document, with only a few exceptions:
 * `window.open()` is disabled.
 
 {% Aside 'caution' %}
-Payment Handler API is only supported in Chrome as of July 2020. However, since
-Chromium based browsers already have the implementation, some of them may expose
-the API in the future. Also, [Mozilla recently announced it's implementing the
-API](https://groups.google.com/g/mozilla.dev.platform/c/gBQp1URD1lE/m/Fswh-5-ZBgAJ).
+Payment Handler API is only supported in Chrome as of January 2023. However,
```

Comment:
Is there a version of Chrome released we are specifically referring to or is this much detail fine?

Reply:
I was trying to express that no other browsers implement the Payment Handler API as of the time of the last update of this page, which is roughly January 2023. Is there a better way to say this?

Chrome has supported Payment Handler API since version 68.

Reply:
Seems alright, I think I was just lost without context.